packages/react-magma-dom/src/components/Stepper/Stepper.tsx

```diff
@@ -45,6 +44,11 @@ export interface StepperProps extends React.HTMLAttributes<HTMLDivElement> {
    * Current step value.
    */
   currentStep: number;
+  /**
+   * Vertical orientation.
+   * @default false
+   */
+  isVertical?: boolean;
```

Comment:
Instead of `isVertical`, we can have an `orientation` prop where `StepperOrientation.horizontal` is default.
```
export enum StepperOrientation {
  horizontal = 'horizontal',
  vertical = 'vertical',
}
```
This would be closer to the pattern of how we handle orientation in other components like ButtonGroup and Tabs.